RELEASING.md

```diff
@@ -0,0 +1,157 @@
+   - The commit and PR title should be 🚀 Release v1.x.y:
+      -`git commit -S -s -m ":rocket: Release v1.x.x"`
+      -`git push -u origin release-notes-1.x.x`
+   - Important! The commit should only contain the release notes file, nothing
+     else, otherwise automation will not work.
+
+- Ask maintainers and release team members to review your pull request.
+
+Once PR is merged following GitHub actions are triggered:
+
+- GitHub action `Create Release` runs following jobs:
+   - GitHub job `push_release_tags` will create and push the tags. This action
+     will also create release branch if its missing and release is `rc` or
+     minor.
+   - GitHub job `create draft release` creates draft release. Don't publish the
+     release until release tag is visible in. Running actions are visible on the
+     [Actions](https://github.com/metal3-io/utility-images/actions)
+     page, and draft release will be visible on top of the
+     [Releases](https://github.com/metal3-io/utility-images/releases).
+     If the release you're making is not a new major release, new minor release,
+     or a new patch release from the latest release branch, uncheck the box for
+     latest release. If it is a release candidate (RC) or a beta release,
+     tick pre-release box.
+   - GitHub jobs `build_keepalived` build release images with the
+     release tag, and push them to Quay. Make sure the release tags are visible in
+     Quay tags pages:
+      - [keepalived](https://quay.io/repository/metal3-io/keepalived?tab=tags)
+     If the new release tag is not available for any of the images, check if the
+     action has failed and retrigger as necessary.
+
+### Release artifacts
+
+We need to verify all release artifacts are correctly built or generated by the
+release workflow. For a release, we should have the following artifacts:
+
+There might be a project specific verify-release.sh in the project directory
+that might automates the following checks:
+
+Git tags pushed:
+
+- Primary release tag: `v0.x.y`
+
+Container images built and tagged at Quay registry:
+
+- [keepalived:v0.x.y](https://quay.io/repository/metal3-io/keepalived?tab=tags)
+
+Files included in the release page:
+
+- Source code
+
+### Make the release
+
+After everything is checked out, hit the `Publish` button your GitHub draft
+release!
+
+## Post-release actions for new release branches
+
+Some post-release actions are needed if new minor or major branch was created.
+
+### Branch protection rules
+
+Branch protection rules need to be applied to the new release branch. Copy the
+settings after the previous release branch, with the exception of
+`Required tests` selection. Required tests can only be selected after new
+keywords are implemented in Jenkins JJB, and project-infra, and have been run at
+least once in the PR targeting the branch in question. Branch protection rules
+require user to have `admin` permissions in the repository.
+
+### Update README.md and build badges
```

Comment:
There are no build badges.

Which also reminds me that probably need to have a periodic jobs for these, as we have not even pinned down the main ubuntu image.

Reply:
release related documentation has been removed.